go/vt/vtgate/router.go

```diff
@@ -374,6 +378,26 @@ func (rtr *Router) execInsertSharded(vcursor *requestContext, route *engine.Rout
 	return result, nil
 }
 
+func (rtr *Router) resolveList(val interface{}, bindVars map[string]interface{}) (vals []interface{}, err error) {
```

Comment:
Not knowing the code too much, this is a bit confusing... maybe comment a bit what this is doing?